pumps/mongo_selective.go

```diff
@@ -199,20 +183,14 @@ func (m *MongoSelectivePump) WriteData(ctx context.Context, data []interface{})
 
 				indexCreateErr := m.ensureIndexes(analyticsCollection)
 				if indexCreateErr != nil {
-					log.WithFields(logrus.Fields{
-						"prefix": mongoSelectivePrefix,
-					}).Error(indexCreateErr)
+					m.log.Error(indexCreateErr)
 				}
 
 				err := analyticsCollection.Insert(dataSet...)
 				if err != nil {
-					log.WithFields(logrus.Fields{
-						"prefix": mongoSelectivePrefix,
-					}).Error("Problem inserting to mongo collection: ", err)
+					m.log.Error("Problem inserting to mongo collection: ", err)
```

Comment:
Again, can we add the name of the collection please

Reply:
Done!